rules/naming/src/PropertyRenamer.php

```diff
@@ -42,13 +47,22 @@ public function __construct(
 
     public function rename(PropertyRename $propertyRename): void
```

Comment:
Maybe a conventions that we use in the rest of code could be applied here.

- If the `Property` is renamed, it should return it.
- If not, the `null`

What do you think?

Reply:
Yeah, makes sense! My brain was in the trap of old logic in the rule.

Reply:
Ready